### ISSUE TITLE
[bug]: hugo pocket opening length depends on pocket width

### ISSUE THREAD

**Issue by anna-puk** [freesewing/freesewing#6004]
### What seems to be the problem? 🤔

The length of the pocket opening for `hugo` depends on the pocket width (the actual width, not the setting). Larger widths result in smaller pocket openings.

Since the pocket width is proportional to hip circumference, 'wide' people are more likely to get unusable pocket openings.

### Design / Plugin / Package 🧐

designs/hugo

### Additional context

The attached screenshot shows how the edge of the pocket opening moves up with a larger pocket width. As a result, the pocket opening gets smaller. (The pattern inspector shows it going from 14.3 to 12.4 cm.)

**Comment by BenJamesBen** [comment]
The pocket opening is what is left after a portion of the side is sewn. Currently, the length of the sewn portion is always 1/3 the width/depth of the pocket.
In `designs/hugo/src/front.mjs`:
```js
  points.pocketTip = points.pocketHem
     .shift(90, points.pocketHem.x / 3)
     .rotate(-30, points.pocketHem)
```

I'm guessing that the length of the sewn portion should instead be based on the height of the pocket.

![Screenshot 2024-02-10 at 5 14 11 PM](https://github.com/freesewing/freesewing/assets/109869956/98c02115-7b98-43f9-90cf-6644b3374840)

**Comment by BenJamesBen** [comment]
Something perhaps like:
```diff
--- a/designs/hugo/src/front.mjs
+++ b/designs/hugo/src/front.mjs
@@ -54,8 +54,9 @@ function hugoFront({
   )
 
   // Ribbing
-  points.cfRibbing = points.cfHem.shift(90, store.get('ribbing'))
-  points.ribbing = points.hem.shift(90, store.get('ribbing'))
+  const ribbingHeight = store.get('ribbing')
+  points.cfRibbing = points.cfHem.shift(90, ribbingHeight)
+  points.ribbing = points.hem.shift(90, ribbingHeight)
 
   // Raglan tip
   const neckOpening = new Path()
@@ -65,14 +66,13 @@ function hugoFront({
   const neckOpeningParts = neckOpening.split(points.raglanTipFront)
 
   // Pocket
-  points.pocketHem = points.cfRibbing.shiftFractionTowards(points.ribbing, options.pocketWidth)
-  points.pocketCf = points.cfHem.shift(
-    90,
-    measurements.hpsToWaistBack * 0.33 + store.get('ribbing')
-  )
+  points.pocketHem = points.cfRibbing
+    .shiftFractionTowards(points.ribbing, options.pocketWidth)
+  const pocketHeight = measurements.hpsToWaistBack * 0.33
+  points.pocketCf = points.cfRibbing.shift(90, pocketHeight)
   points.pocketTop = new Point(points.pocketHem.x, points.pocketCf.y)
   points.pocketTip = points.pocketHem
-    .shift(90, points.pocketHem.x / 3)
+    .shift(90, pocketHeight / 3)
     .rotate(-30, points.pocketHem)
   points.pocketTopCp = utils.beamsIntersect(
     points.pocketTop,
```

**Comment by joostdecock** [comment]
I've fixed this in #6010 with:

```diff
diff --git a/designs/hugo/src/front.mjs b/designs/hugo/src/front.mjs
index 8b25c5c68cf..19979961a6e 100644
--- a/designs/hugo/src/front.mjs
+++ b/designs/hugo/src/front.mjs
@@ -72,7 +72,7 @@ function hugoFront({
   )
   points.pocketTop = new Point(points.pocketHem.x, points.pocketCf.y)
   points.pocketTip = points.pocketHem
-    .shift(90, points.pocketHem.x / 3)
+    .shift(90, points.pocketTop.dy(points.pocketHem) / 3.5)
     .rotate(-30, points.pocketHem)
   points.pocketTopCp = utils.beamsIntersect(
     points.pocketTop,
```

Before the fix, you can see the upward trend of the bottom of the pocket opening when we sample the hip measurement:

![Screenshot from 2024-02-11 12-48-03](https://github.com/freesewing/freesewing/assets/1708494/1c360231-2814-49de-936d-98b29f51b13b)

After the fix, the pocket opening remains the same:

![Screenshot from 2024-02-11 12-50-18](https://github.com/freesewing/freesewing/assets/1708494/9438f96f-80bf-4302-ba2a-aa8b792ad8d5)
